server/src/main/java/com/skillmagnet/Course/CourseController.java

```diff
@@ -20,75 +20,82 @@ public class CourseController {
 
     /**
      * Retrieves specific course by id
+     *
```

Comment:
:memo: there's lots of miscellaneous spaces and formatting that got applied from my autoformatter